FinniversKit/Sources/Components/SelfDeclaration/SelfDeclarationView.swift

```diff
@@ -1,61 +1,61 @@
 //
-//  Copyright © FINN.no AS, Inc. All rights reserved.
+//  SelfDeclarationView.swift
+//  FinniversKit
 //
 
-public class SelfDeclarationView: UIView {
+import SwiftUI
 
-    // MARK: - Private properties
+public struct SelfDeclarationView: View {
 
-    private lazy var introductionLabel: Label = {
-        let label = Label(style: .body, withAutoLayout: true)
-        label.numberOfLines = 0
-        return label
-    }()
+    public let vm: SelfDeclarationViewModel
 
-    private lazy var stackView: UIStackView = {
-        let stackView = UIStackView(withAutoLayout: true)
-        stackView.axis = .vertical
-        stackView.spacing = .spacingL
-        return stackView
-    }()
-
-    // MARK: - Init
-
-    public override init(frame: CGRect) {
-        super.init(frame: frame)
-        setup()
-    }
-
-    required init?(coder: NSCoder) { fatalError() }
-
-    // MARK: - Setup
-
-    private func setup() {
-        stackView.addArrangedSubview(introductionLabel)
-        stackView.setCustomSpacing(.spacingXL, after: introductionLabel)
-        addSubview(stackView)
-        stackView.fillInSuperview()
+    public init(vm: SelfDeclarationViewModel) {
```

Comment:
For consistency: `vm` should be changed to `viewModel` 🙂

Reply:
Fixed!